src/inkbird_ble/parser.py

```diff
@@ -70,6 +72,9 @@ def _start_update(self, service_info: BluetoothServiceInfo) -> None:
             dev_type, _ = bbq_data
             self.set_device_name(f"{local_name} {short_address(address)}")
             self.set_device_type(f"{local_name[0]}{dev_type[1:]}")
+        elif IAMT1_SERVICE_UUID in service_info.service_uuids:
+            self.set_device_name(f"{local_name} {short_address(address)}")
```

Comment:
I wouldn't expect this to work. The T1 I have doesn't advertise this service (or any service) so I would expect `service_uuids` to be empty here. Did this work?

The manufacturer data (0x3154 "AC-6200a135990\0" or maybe they meant "1TAC-..."?) feels like it may be a more reliable identifier (plus the device name?)

(FFE0 is also not an assigned service. It's a service that a lot of devices squat on without registering. So its existence doesn't tell you anything other than the device developer was sloppy, and possibly just following a random tutorial or copied HOPERF modules.)